code/game/objects/items/plushes.dm

```diff
@@ -800,3 +800,51 @@
 	icon_state = "donkpocket"
 	attack_verb_continuous = list("donks")
 	attack_verb_simple = list("donk")
+
+/obj/item/toy/plush/Zorp //It's supposed to be a wet owl, but I call it Zorp
+	name = "Wet owl plush"
+	desc = "Plush is an owl who has seen some shit"
```

Comment:
```suggestion
	desc = "This plush has seen some shit."
```

Reply:
Oh, if the strange wording is a reference to a translation then leave it I suppose